app/views/foreman_ansible/job_templates/convert_to_rhel.erb

```diff
@@ -20,11 +20,25 @@ template_inputs:
   advanced: false
   value_type: plain
   hidden_value: false
+- name: ELS
+  required: true
+  input_type: user
+  description: Explicitly recognize the system as els, utilizing els repos.
```

Comment:
```suggestion
  description: By default, the job converts a RHEL 7-based system to a supported RHEL 7 system with the latest security patches and updates via the Extended Lifecycle Support (ELS).

If you plan to upgrade to RHEL 8 right after the conversion, you may opt not to use the ELS subscription. Note that the conversion and the subsequent upgrade without an ELS subscription come with a limited support scope per the [Convert2RHEL Support Policy](https://access.redhat.com/support/policy/convert2rhel-support) and the [In-place upgrade Support Policy](https://access.redhat.com/support/policy/ipu-support).
```
I'm not sure if links are supported.
However, this option applies to conversions to RHEL 7 only so it's better to make it clear what it's about.

Reply:
@Lennonka I agree with the suggestion, but the second part is a better fit, I would leave it for the documentation. WDYT?

Reply:
Also, I agree with you about EL7 conversions, but this JT also supports EL8 conversions, and currently, this ELS option with this description seems to be specific to EL7 only

Reply:
I would remain neutral about the RHEL version in the description because RHEL 8 will become ELS at some point as well. And IMHO what the user does with the system after conversion is out of scope for this context ("If you plan...").

```suggestion
  description: Use an Extended Lifecycle Support (ELS) add-on subscription.
  If you plan to upgrade your system to a supported RHEL version after the conversion, ELS is not required.
```
I'm not sure if we have to include the note about supportability.